deluge/ui/gtkui/aboutdialog.py

```diff
@@ -7,29 +7,28 @@
 # See LICENSE for more details.
 #
 
-import gtk
-import pygtk
-
+import gi
+from gi.repository import GdkPixbuf, Gtk
 from deluge.common import get_pixmap, get_version, open_url_in_browser
 from deluge.ui.client import client
 from deluge.ui.gtkui.common import get_deluge_icon
 
-pygtk.require('2.0')
+gi.require_version('Gtk', '3.0')
 
 
 class AboutDialog:
     def __init__(self):
         def url_hook(dialog, url):
             open_url_in_browser(url)
-        gtk.about_dialog_set_url_hook(url_hook)
```

Comment:
from what i can tell no it doesnt. However i think we need a workaround of sorts because clicking the website link it pops up a windows saying no application to handle this or something like that. However the website items in the help menu open the browser just fine.